googlemock/docs/cheat_sheet.md

```diff
@@ -235,10 +235,10 @@ several categories:
 
 #### Wildcard
 
-Matcher                     | Description
-:-------------------------- | :-----------------------------------------------
-`_`                         | `argument` can be any value of the correct type.
-`A<type>()` or `An<type>()` | `argument` can be any value of type `type`.
+| Matcher                     | Description                                      |
```

Comment:
You're second commit called "Fix typo in tables in docs" does some changes which don't change anything. In Markdown you may build up a table row both with or without enclosing pipes, i.e. ` ABC | XYZ ` and ` | ABC | XYZ |` are synonyms. I certainly wouldn't call it a typo.
Note: this relates to other places in this review as well.

Reply:
Thank you for your comment !

Yeap, you are right, They both work well !
I just did it to let them use the same syntax as the others in the document!

Reply:
Cool! You're welcome. Could you possibly rename the commit as I mentioned?

Reply:
I'm sorry for the late response!
Could you please review again whether commit's name is proper or not~?